README.md

```diff
@@ -1,20 +1,25 @@
 # OVAL visualization as graph
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/quality-score.png?b=codeStyle)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=codeStyle) [![Code Coverage](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/coverage.png?b=codeStyle)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=codeStyle) [![Build Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/build.png?b=codeStyle)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/build-status/codeStyle) [![Code Intelligence Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/code-intelligence.svg?b=codeStyle)](https://scrutinizer-ci.com/code-intelligence)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Code Coverage](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Build Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/build.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/build-status/master) [![Code Intelligence Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/code-intelligence.svg?b=master)](https://scrutinizer-ci.com/code-intelligence)
 
 Understanding result in the blink of an eye
 
 ### How use?
 ```
 Command:
-    python3 run.py arf-file.xml id-rule
+    python3 run.py ARF-file.xml id-rule
 Example:
     python3 run.py data/ssg-fedora-ds-arf.xml xccdf_org.ssgproject.content_rule_disable_host_auth
 ```
-## Requires:
+* *ARF-file.xml* -  ARF xml report from OpenSCAP scan. 
+* *id-rule*  - Rule ID to be visualized. You can input part of ID rule or use regular expresion,but if you use in regular expresion brackets. You must put regular expression betwen quotation marks.
+
+It open web browser with graph. Default is Firefox. If Firefox not installed it open default web browser in OS.  
+![demo-screenshot](demo-screenshot.png "demo-screenshot")
+* ### Minimal requires:
```

Comment:
Great screenshot! However, the language needs to get better - there are many mistakes s.a. "it open**s**", "minimal require**ment**s" etc. `README` is the first thing people see, so make it great.